wpilibc/src/main/native/cpp/frc2/command/SynchronousPIDCommand.cpp

```diff
@@ -0,0 +1,57 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "frc2/command/SynchronousPIDCommand.h"
+
+using namespace frc2;
+
+SynchronousPIDCommand::SynchronousPIDCommand(
+    PIDController controller, std::function<double()> measurementSource,
+    std::function<double()> setpointSource,
+    std::function<void(double)> useOutput,
+    std::initializer_list<Subsystem*> requirements)
+    : m_controller{controller},
+      m_measurement{measurementSource},
```

Comment:
The implementation in inconsistent with whether the passed-in std::functions are moved or copied into the member variables (e.g. StartEndCommand.cpp moves, while this copies).